docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc

```diff
@@ -214,7 +214,40 @@ For more information about devfile contents requirements, see the "What is outer
 +
 **Solution:** Select a predefined runtime type from the **Review and configure for deployment** view. Make sure you meet all of the {ProductName} <<repository requirements>> for the runtime you select, or configure your own <<custom build and deployment instructions>>.
 
-[role="_additional-resources"]
-.Additional resources
```

Comment:
@jc-berger don't throttle me lol, but [Burr just asked Chloe yesterday](https://redhat-internal.slack.com/archives/C04PZ7H0VA8/p1685542679008949) to add this content back in ASAP. including it in Add'l resources is obv not a make-or-break blocker, but at some point, check w/Chloe and see whether this CLI content is indeed available now to link to. FYI